android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/DDMFormViewConfig.kt

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ * <p>
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ * <p>
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.mobile.screens.viewsets.defaultviews.ddm.form
+
+/**
+ * @author Paulo Cruz
+ */
+data class DDMFormViewConfig @JvmOverloads constructor(var autoloadDraftEnabled: Boolean = true,
+	var autosaveDraftEnabled: Boolean = true, var syncFormTimeout: Long = 500)
```

Comment:
this `syncFormTimeout` can be changed with the constant you already created called DEFAULT_TIMEOUT?